zerver/tests/test_push_notifications.py

```diff
@@ -1546,8 +1546,8 @@ def test_when_kind_is_not_none(self) -> None:
                                           kind=PushDeviceToken.APNS)
         self.assertEqual(count, 2)
 
-class TestPushApi(ZulipTestCase):
-    def test_push_api(self) -> None:
```

Comment:
I have split `test_push_api()` into `test_push_api_error_handling()` and `test_push_api_add_and_remove_device_tokens()` to test adding and removing tokens in detail for both local and remote servers. Please tell me if this idea doesn't sound good.